apiary.apib

```diff
@@ -372,7 +372,7 @@ We consider the following changes to be backwards-incompatible:
 We list all backwards-compatible additions here. These are currently available in all published versions.
 (There is also a [list of backwards-incompatible upgrades](#changelog) available, but those only apply if you [upgrade your API version](#upgrading-your-api-version).)
 
-#### Latest
+#### May 2020
```

Comment:
Can you add a new latest?

Reply:
I could, but there would be nothing in there?

Reply:
Yeah let's add it again when there are new changes